src/scss/components/banner/_banner-quote-text.scss

```
@@ -0,0 +1,86 @@
+@charset 'UTF-8';
+
+// Top banner styles
+.irsr-wrapper--banner-quote-text {
+  .su-hero {
+    display: block;
+    min-height: 220px;
+  
+    .su-hero__media {
+      min-height: auto;
+      position: absolute;
+  
+      img {
+        opacity: 0.4;
+      }
+    }
+  
+    .su-hero__card {
+      background: transparent;
+      border: 0;
+      box-shadow: none;
+      left: auto;
+      text-align: center;
+      max-width: unset;
+      margin: 0;
+      bottom: unset;
+  
+      .su-card__contents {
+        @include responsive-spacing('padding-top', 4);
+        @include responsive-spacing('padding-bottom', 5);
+
+        color: $su-color-white;
+        width: 75%;
+        margin: 0 auto;
+        min-height: 220px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+
+        @include grid-media-max('sm') {
+          width: 100%;
+        }
+
+        :first-child {
+          margin-top: 0;
+        }
+  
+        h2 {
+          @include type-a;
+          font-family: $su-font-serif;
+        }
+  
+        p,
+        blockquote {
+          @include big-paragraph;
+          margin-right: 0;
+          margin-left: 0;
+
+          &:first-of-type {
+            &::before {
+              display: block;
```

```suggestion
              @include modular-spacing('margin-bottom', 1);
              display: block;
```
This adds a bit more spacing between the open-quote and quote text for larger screens